### DISCUSSION TITLE
Some sites using EME

### DISCUSSION THREAD

**Discussion by liedekef** [liedekef/events-made-easy#11] [Show and tell]
https://lkc-lueneburg.de/termine/
http://www.csga.cz/kalendar-turnaju/
https://www.bonheidensehondenvrienden.be/lidmaatschap-bhv/
https://www.theaterdejavu.be/wordpress/

**Comment by bobbybosler** [comment]
Also,

https://fallsbaptist.org/
https://baptistcollege.org/
https://minutemenministries.org/
https://colaclash.org/
https://thewar.team/
https://warsf.org/
https://bobbybosler.org/itinerary/ (in development)

**Comment by TRILOS** [comment]
https://www.labournet.de/termine/
with EMEFS
https://www.labournet.de/termin-oder-veranstaltung-eingeben/

**Comment by f14m07** [comment]
Association website for sailing with EME and EMEFS https://vogue-et-reve.fr
